package.json

```diff
@@ -73,6 +73,7 @@
     "prom-client": "^11.5.3",
     "qs": "^6.9.1",
     "request": "^2.88.1",
+    "request-promise-native": "^1.0.9",
```

Comment:
Is there a reason you added this particular dependency? I checked on npm and it is deprecated.
We can use another library: got, axios or node-fetch are popular choices.

Reply:
It's the promises version of the `request` library, which is also deprecated, but at least it means we don't have two different HTTP libraries in there.

Reply:
Or rather, it's a promises + async-compatible wrapper around `request`

Reply:
Fair point. Replacing `request` and the wrapper is out of scope for this change.